cloud/azure/main.tf

```diff
@@ -153,23 +153,23 @@ resource "azurerm_postgresql_server" "civiform" {
   location            = azurerm_resource_group.rg.location
   resource_group_name = azurerm_resource_group.rg.name
 
-  administrator_login          = "psqladmin"
-  administrator_login_password = "H@Sh1CoR3!"
+  administrator_login          = var.postgres_admin_login
+  administrator_login_password = var.postgres_admin_password
 
   // fqdn civiform-db.postgres.database.azure.com
 
-  sku_name   = "GP_Gen5_4"
-  version    = "11"
-  storage_mb = 5120
+  sku_name   = var.postgres_sku_name
+  version    = var.postgres_version
+  storage_mb = var.postgres_storage_mb
 
-  backup_retention_days        = 7
+  backup_retention_days        = var.postgres_backup_retention_days
   geo_redundant_backup_enabled = false
   auto_grow_enabled            = true
 
   public_network_access_enabled = false
 
   ssl_enforcement_enabled          = true
-  ssl_minimal_tls_version_enforced = "TLS1_2"
+  ssl_minimal_tls_version_enforced = var.postgres_ssl_minimal_tls_version_enforced
```

Comment:
suggestion(blocking): We probably want to hard code this to "TLS1_2" since we're storing sensitive data and TLS 1.1 and 1.0 are insecure and deprecated. See https://www.digicert.com/blog/depreciating-tls-1-0-and-1-1